keras_cv/layers/preprocessing/grayscale.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+import tensorflow.keras.layers as layers
+from tensorflow.keras import layers, backend
+
+
+class Grayscale(layers.Layer):
+    """Grayscale class for transforming RGB image to Grayscale image. The expected images 
```

Comment:
Update to:

Grayscale is a preprocessing layer that transforms RGB images to Grayscale images.  

Input images should have values in the range of [0, 255].

Reply:
Done.